content/docs/other-guides/usage-reporting.md

```diff
@@ -20,24 +20,34 @@ command only restarts spartakus with reportUsage set to `false`
 kubectl -n ${NAMESPACE} delete deploy -l app=spartakus
 ```
 
-**Reporting usage data is one of the most significant contributions you can make to Kubeflow; so please consider turning it on.** This data
-allows us to improve the project and helps the many companies working on Kubeflow justify continued investment.
-
-You can improve the quality of the data by giving each Kubeflow deployment a unique id by editing `spartakus.yaml` in your `KF_APP` directory:
-
-```
-apiVersion: v1
-data:
-  usageId: "your usage id"
-kind: ConfigMap
-metadata:
-  labels:
-    kustomize.component: spartakus
-  name: spartakus-parameters
-  namespace: kubeflow
-```
-
-Then deploy your changes:
-```
-kubectl apply -f spartakus.yaml`
+**To prevent Spartakus from being deployed; users should do the following before running `kfctl apply`** 
+- Edit ${KFAPP}/app.yaml
+- Delete the entry in KfDef.Spec.Applications for spartakus
+```
+  - kustomizeConfig:
+      parameters:
+      - initRequired: true
+        name: usageId
+        value: <randomly-generated-id>
+      - initRequired: true
+        name: reportUsage
+        value: "true"
+      repoRef:
+        name: manifests
+        path: common/spartakus
+    name: spartakus
+```
+- Delete the entry in KfDef.Spec.ComponentParams for spartakus
```

Comment:
This section doesn't render properly. See preview https://deploy-preview-1085--competent-brattain-de2d6d.netlify.com/docs/other-guides/usage-reporting/

Reply:
Done，please review that.  Thanks again.

Reply:
The code-formatted section still doesn't show up properly on the page. You can see the problem in the preview here: https://deploy-preview-1085--competent-brattain-de2d6d.netlify.com/docs/other-guides/usage-reporting/

To fix the problem, add an empty line before the start of the code-formatted block. That is, add an empty line after the first "spartakus" on line 43.